### ISSUE TITLE
Wrong colors of emoticon

### ISSUE THREAD

**Issue by martin-77** [nextcloud/talk-desktop#914]
----------------------------System report----------------------------
Nextcloud Talk Desktop version 1.0.0-rc.1
- Built with Nextcloud Talk version 20.0.2

Connected to:
- Server address: https://sub.fqdn.de
- Nextcloud Server version 30.0.2
- Nextcloud Talk version 20.0.2

OS: Windows_NT 10.0.26100 (Windows 11 Pro)
----------------------------System report----------------------------

When receiving a "red heart" as reaction to a message, the heart in the client shows up as blue heart.

Why is this so bad and worth the issue?

In Germany, we have a racist party in our parliament, which followers use a blue heart to identify each other as "fans" of that fu..... party.

Had a dispute to a collegue caused by the usage of the blue heart. She argued, that she send a red heart - I checked browser and talk app - she send a red heart.

Please: fix this, no one wants to support racists in an way.

Thanks

Martin

![Image](https://github.com/user-attachments/assets/dd544d06-9044-4140-9513-c741530c3841)

**Comment by ShGKme** [comment]
Emoji picker supports all emoji hearts, and I'm not aware of any rendering issues with hearts on Windows. Do you have any non-standard fonts/themes installed on Windows?

![Image](https://github.com/user-attachments/assets/300eca72-12fb-4a6a-bc2f-71b6132e7e33)

![Image](https://github.com/user-attachments/assets/509ef22c-2f3e-4ccd-9f19-85c6269f37da)

Could you using devtools (CTRL+SHIFT+I, CTRL+SHIFT+C) find this button on the app and then copy the emoji and send here? If this is actually not a blue emoji symbol, then there might be a font issue.

![Image](https://github.com/user-attachments/assets/dcf2a156-ffd3-45dc-95c8-673be0ca55c4)

**Comment by ShGKme** [comment]
A screenshot with an emoji picker on "heart" search could also help.

**Comment by martin-77** [comment]
Thanks for your hints and quick reaction.

Here a screenshot from dev-tools

![Image](https://github.com/user-attachments/assets/7e0985f5-20d6-4a04-a9ac-e69fddfb49f4)

**Comment by martin-77** [comment]
> A screenshot with an emoji picker on "heart" search could also help.

The Message was send from android talk app, nothing special. 
In the apps from Android and iOS, the heart is shown in red.

When switching from dark to white theme on talk desktop, the heart is black-ish

![Image](https://github.com/user-attachments/assets/be436a07-79f9-42f7-9254-82097558ece0)

And damn - when switching from white to dark theme in Nextcloud talk - the heart is blue again...

![Image](https://github.com/user-attachments/assets/c2260333-dfc8-486e-b041-a92202ab890c)

Seems to be an Issue on theming? right? or talk? clueless...

**Comment by nickvergessen** [comment]
No, it's an uncolored default heart ascii character. So just picking up the normal text color, all as intended.
But that is basically why my colleague asked for a screenshot of the emoji picker.

![Image](https://github.com/user-attachments/assets/10d33aa6-9d6e-4245-ad1b-74d0728b1263)

You can see the 3D variant (top left) called `heart` and the "flat" (bottom right) called `hearts` which refers to the symbol from card games. Ref

3D | flat
---|---
❤️|♥️

**Comment by nickvergessen** [comment]
Maybe we can try to tell systems to prefer the emoji font for that part, instead of the normal text font

**Comment by nickvergessen** [comment]
- Quick fix in https://github.com/nextcloud/spreed/pull/13842

**Comment by martin-77** [comment]
Thank you soooo much!
